docs/developer-guide/governance.md

```diff
@@ -55,9 +55,9 @@ Reviewers have additional responsibilities over and above those of a Committer.
 Reviewers fulfill all requirements of Committers, and also:
 
 * Contributions can be reviewed by other Reviewers, but this is not explicitly required.
-* Are not required to submit pull requests for changes, however, they are strongly encouraged to do so when the change is non-trivial.
-* May merge their own pull requests once they have collected the feedback they deem necessary.
-* Can label issues as they are submitted (but must not add the "accepted" label).
+* May merge external pull requests for accepted issues upon reviewing and approving the changes.
+* May merge their own pull requests once they have collected the feedback they deem necessary. (No pull request should be merged without at least one Committer/Reviewer comment stating they've looked at the code.)
```

Comment:
I'd rather keep this generic so it makes sense for all repos. I kind of consider you the project lead for the website, so you have superpowers there.

I'd purger even blocking/urgent changes have a review as well. We can always make exceptions in certain cases, but I think a second set of eyes is always preferable.